README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
 </settings>
 ```
 
-### More information
+#### More information
+
 - https://central.sonatype.org/pages/ossrh-guide.html
+
+## Configuration options
+
+The `UnleashConfig` class exposes a set of builder methods to configure your Unleash client. The available options are listed below with a description of what they do. For the full signatures, take a look at the [`UnleashConfig` class definition](src/main/java/io/getunleash/util/UnleashConfig.java).
+
+| Method name                                | Description                                                                                                                                                         | Required | Default value                        |
+|--------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------|
+| `appName`                                  | The name of the application as shown in the Unleash UI. Registered applications are listed on the Applications page.                                                | Yes      | `null`                               |
+| `backupFile`                               | The path to the file where [local backups](#local-backup) get stored.                                                                                               | No       | `"java.io.tmpdir/unleash-repo.json"` |
```

Comment:
Is this the correct default backup file location?

Reply:
I don't think so, the backup file name contains the appName as well, from the source code (tweaked for clarity): 
```
System.getProperty("java.io.tmpdir") + File.separatorChar + "unleash-" + appName + "-repo.json";
```